src/Microsoft.Health.Dicom.Core/Features/CustomTag/GetCustomTagsService.cs

```diff
@@ -30,14 +32,28 @@ public async Task<GetCustomTagResponse> GetCustomTagAsync(string tagPath, Cancel
         {
             string internalTagPath = _dicomTagParser.ParseFormattedTagPath(tagPath);
```

Comment:
should the path be normalized at first? 
e.g: 0008002a is a valid input, but SQL SP would not return correct value due to the lowercase 'a'. 
Recommend to convert to DicomTag at first, then call Dicomtag.GetPath to normalize. (https://github.com/microsoft/dicom-server/blob/754db8d60dfa3bc66c10b073c0e3846dc9b10237/src/Microsoft.Health.Dicom.Core/Extensions/CustomTagEntryExtensions.cs#L26)